.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,3 +3,35 @@ To be copied to the [draft changelog](https://github.com/DataBiosphere/toil/wiki
 
  * Whatsits now frobnicated
 
+## Reviewer Checklist
+
+<!-- To be kept in sync with docs/contributing/checklist.rst -->
+
+ * [ ] Make sure it is coming from `issues/XXXX-fix-the-thing` in the Toil repo, or from an external repo.
+    * [ ] If it is coming from an external repo, make sure to pull it in for CI with:
+        ```
+        contrib/admin/test-pr otheruser theirbranchname issues/XXXX-fix-the-thing
+        ```
+    * [ ] If there is no associated issue, [create one](https://github.com/DataBiosphere/toil/issues/new).
+* [ ] Read through the code changes. Make sure that it doesn't have:
+    * [ ] Addition of trailing whitespace.
+    * [ ] New variable or member names in `camelCase` that want to be in `snake_case`.
+    * [ ] New functions without [type hints](https://docs.python.org/3/library/typing.html).
```

Comment:
To be added to the Toil Makefile
```
mypy --cobertura-xml-report . src/toil
diff-cover  cobertura.xml
```
Will alert you to new functions without typehints